Makefile

```diff
@@ -86,8 +86,10 @@ install: all
 	$(MKDIR) $(SHAREDIR)
 	$(CP) script xsd xslt vendor lib.sh $(SHAREDIR)
 	$(MKDIR) $(BINDIR)
-	sed '/^SHAREDIR=/c SHAREDIR="$(SHAREDIR)"' bin/ocr-transform.sh > $(BINDIR)/ocr-transform
-	sed '/^SHAREDIR=/c SHAREDIR="$(SHAREDIR)"' bin/ocr-validate.sh > $(BINDIR)/ocr-validate
+	sed '/^SHAREDIR=/c SHAREDIR="$(SHAREDIR)"' bin/ocr-transform.sh | \
+	  sed "s/[$$](git describe)/$$(git describe)/" > $(BINDIR)/ocr-transform
```

Comment:
Is the double $$ in the regex list needed?

Reply:
It is needed because `make` normally interprets the $ as an indicator for a make variable. `sed` only sees a single $, likewise the shell, so the line is executed as `sed "s/[$](git describe)/$(git describe)/"` and will replace the textual string `$(git describe)` by the result of that shell expression.